### ISSUE TITLE
Two more required packages for build and make for nsfminer

### ISSUE THREAD

**Issue by DJrunkie** [no-fee-ethereum-mining/nsfminer#28]
I just wanted to note that I was receiving errors with the build and make but resolved them by installing these two packages for Fedora 32.

fifechan-opengl-devel.x86_64 
libstdc++-static.x86_64 

These were the two errors I was receiving

<GL/gl.h> not found
cannot find -lstdc++

Just wanted to mention those if someone else has any issues as they were not in the list of required packages here https://github.com/no-fee-ethereum-mining/nsfminer/blob/master/docs/BUILD.md .

However its up and running now, currently mining eth on Cuda 11 with a 1080ti.

 miner Job: 5f7eac9e blk: 11734154 us1.ethermine.org:4444
 miner 0:03 A3 33.74 Mh - cu0 33.74
 miner 0:04 A3 33.31 Mh - cu0 33.31

Cheers!

**Comment by DJrunkie** [comment]
Also sorry to post again as this is not related to any issue but wanted to note I was able to increase my hashrate by about 30% using OhGodAnETHlargementPill, which afaik only works with 1080, 1080ti and Titan XP because they have gddr5x memory. 

 miner 0:08 A8 43.96 Mh - cu0 43.96
 miner Job: 813c54bb blk: 11734378 us1.ethermine.org:4444
 miner Job: 62d6238b blk: 11734378 us1.ethermine.org:4444
 miner 0:09 A8 43.77 Mh - cu0 43.77

I had to enable iomem=relaxed for my kernel boot options to get OhGodAnETHlargementPill to work.   However now I'm getting between 41-44 Mh now on a 1080ti which appears to be quite impressive.
